tool/travis.sh

```diff
@@ -14,31 +11,35 @@ if [ "$#" == "0" ]; then
   exit 1
 fi
 
+EXIT_CODE=0
+
 pushd $PKG
-pub upgrade
+pub upgrade || EXIT_CODE=$?
```

Comment:
We don't want a failed `pub upgrade` to fail fast?

Reply:
well... no probably not lol :D, will fix

Reply:
done